addons/web/static/src/core/network/download.js

```diff
@@ -545,6 +549,10 @@ download._download = (options) => {
         xhr.onerror = () => {
             reject(new ConnectionLostError());
         };
-        xhr.send(data);
+        if (data) {
+            xhr.send(data);
+        } else {
+            xhr.send();
+        }
```

Comment:
or maybe always create a (potentially empty) formdata and always post it? what happens when you post an empty formdata with a GET?